pkgpanda/__init__.py

```diff
@@ -678,9 +712,22 @@ def symlink_all(src, dest):
 
         active_buildinfo_full = {}
 
+        dcos_service_configuration = self._get_dcos_configuration_template()
+
         # Building up the set of users
         sysusers = UserManagement(self.__manage_users, self.__add_users)
 
+        def _get_service_names(_dir):
+            service_files = list(map(os.path.basename,
+                                     glob.glob(_dir + "/dcos.target.wants/*.service") +
+                                     glob.glob(_dir + "/dcos.target.wants_master/*.service") +
+                                     glob.glob(_dir + "/dcos.target.wants_slave/*.service") +
+                                     glob.glob(_dir + "/dcos.target.wants_slave_public/*.service")))
+            if not service_files:
```

Comment:
I'd also rather always return a list (which may be empty) than none in this case. Like minimizing the number of types when possible.